docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -174,6 +174,17 @@ The `RefreshScope` is a bean in the context and has a public  `refreshAll()` met
 The `/refresh` endpoint exposes this functionality (over HTTP or JMX).
 To refresh an individual bean by name, there is also a `refresh(String)` method.
 
+To expose the `/refresh` endpoint you will need to add following configuration to your application.
+
+[source,yaml]
+----
+management:
+  endpoints:
+    web:
+      exposure:
+        include: refresh
+----
+
 NOTE: `@RefreshScope` works (technically) on an `@Configuration` class, but it might lead to surprising behavior.
 For example, it does *not* mean that all the `@Beans` defined in that class are themselves in `@RefreshScope`.
```

Comment:
Remove the highlight. It interferes with reading. People's eyes jump there first, which forces them to read the sentence twice. The sentence is well enough constructed to not need the highlight. If you really want it to stand out, make the NOTE into a CAUTION.